lib/hammer_cli/ssloptions.rb

```diff
@@ -0,0 +1,27 @@
+module HammerCLI
+  def self.get_ssl_options
+    settings = HammerCLI::Settings
+    logger = Logging.logger['SSLoptions']
```

Comment:
Do you think you could move `logger` and `settings` to the function constructor and make it injectable dependencies? It would make testing easier and more straightforward.